Library/Homebrew/cmd/help.rb

```diff
@@ -89,4 +93,15 @@ def command_help(path)
       end.join.strip
     end
   end
+
+  def command_help_cmd(path)
+    path_str = path.to_s
+    cmd = path_str.match(%r{Homebrew/cmd/(.*).rb})[1]
+    class_name = "#{cmd.capitalize}Command"
+    class_instance = Object.const_get("Homebrew::#{class_name}").new
```

Comment:
You could save a bit of string manipulation here by doing `Homebrew.const_get(class_name)`.